scene/gui/rich_text_label.cpp

```diff
@@ -4157,6 +4157,18 @@ int RichTextLabel::get_content_height() const {
 	return total_height;
 }
 
+int RichTextLabel::get_content_width() const {
+	int total_width = 0;
+	if (main->lines.size()) {
+		for (int i = 0; i < main->lines.size(); i++) {
+			if (total_width < main->lines[i].offset.x + main->lines[i].text_buf->get_size().x) {
+				total_width = main->lines[i].offset.x + main->lines[i].text_buf->get_size().x;
+			}
```

Comment:
```suggestion
			total_width = MAX(total_width, main->lines[i].offset.x + main->lines[i].text_buf->get_size().x);
```
It will take less space, and be more readable.

Reply:
Done.